src/main/java/core/basesyntax/strategy/Main.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.strategy;
```

Comment:
```suggestion
package core.basesyntax;
```
use strategy package for handlers and Strategy class